tests/models/bert/test_tokenization_bert_tf.py

```diff
@@ -4,15 +4,15 @@
 
 from transformers import AutoConfig, TFAutoModel, is_tensorflow_text_available, is_tf_available
 from transformers.models.bert.tokenization_bert import BertTokenizer
-from transformers.testing_utils import require_tensorflow_text, slow
+from transformers.testing_utils import require_tensorflow_text, require_tf, slow
 
 
-if is_tensorflow_text_available():
-    from transformers.models.bert import TFBertTokenizer
-
 if is_tf_available():
     import tensorflow as tf
 
+    if is_tensorflow_text_available():
```

Comment:
@sgugger As `is_tensorflow_text_available` already contains the check for TF, should I revert the change here? Or it's fine to keep it this way?

Reply:
- In current `main` branch, `is_tensorflow_text_available` condition is not inside `is_tf_available`. In past CI image build dockerfile, we install `.[dev]` then uninstall `tensorflow`, so `tensorflow_text` is there but `tensorflow` is removed. And this causes some tensorflow_text-related tests failed (`TFBertTokenizer` file will import `tensorflow`)

- In the 1st version of the PR, I keep `is_tensorflow_text_available` as it is, and use the combination of `is_tensorflow_text_available() and is_tensorflow_available()`. **The change at these 2 lines was necessary at that time**.

- @sgugger said we should instead change the definition of `is_tensorflow_text_available` to include `is_tensorflow_available` directly. After applying that suggestion, **we no longer need to change the 2 lines here**.

Reply:
You can unindent this block if you want but honestly we don't really care. The tensorflow import needs to stay as it's used in the file.